src/reversion_rest_framework/mixins.py

```diff
@@ -15,11 +16,24 @@ class BaseHistoryModelMixin:
 class HistoryOnlyMixin(BaseHistoryModelMixin):
     @action(detail=True, methods=['GET'], name='Get History')
     def history(self, request, pk=None):
-        object = self.get_object()
-        versions = Version.objects.get_for_object(object)
+        instance = self.get_object()
+        versions = Version.objects.get_for_object(instance).order_by(
+            "revision__created_date"
+        )
+        page = self.paginate_queryset(versions)
```

Comment:
following https://stackoverflow.com/questions/57222981/how-to-paginate-within-an-action-in-django-rest-framework

Reply:
Nice!